aurora/main.tf

```diff
@@ -44,26 +48,26 @@ resource "aws_db_parameter_group" "aurora_mysql" {
 }
 
 resource "aws_rds_cluster" "aurora" {
-  cluster_identifier              = "${var.project}-${var.environment}${var.tag}-aurora"
-  master_username                 = "${var.rds_username}"
-  master_password                 = "${var.password}"
-  backup_retention_period         = "${var.backup_retention_period}"
-  skip_final_snapshot             = "${var.skip_final_snapshot}"
+  cluster_identifier              = "${var.project}-${var.environment}${var.tag}-aurora}"
+  master_username                 = var.rds_username
+  master_password                 = var.password
+  backup_retention_period         = var.backup_retention_period
+  skip_final_snapshot             = var.skip_final_snapshot
   final_snapshot_identifier       = "${var.project}-${var.environment}${var.tag}-aurora-final-${md5(timestamp())}"
-  availability_zones              = ["${data.aws_availability_zones.available.names}"]
-  db_subnet_group_name            = "${aws_db_subnet_group.aurora.id}"
-  vpc_security_group_ids          = ["${aws_security_group.sg_aurora.id}"]
-  storage_encrypted               = "${var.storage_encrypted}"
-  apply_immediately               = "${var.apply_immediately}"
-  db_cluster_parameter_group_name = "${var.cluster_parameter_group_name}"
-  engine                          = "${var.engine}"
-  engine_version                  = "${var.engine_version}"
-  enabled_cloudwatch_logs_exports = ["${var.enabled_cloudwatch_logs_exports}"]
+  availability_zones              = [data.aws_availability_zones.available.names]
```

Comment:
I think you should remove the list notation here, the `data.aws_availability_zones.available.names` is already a list afaik